File: C programming/Chapter-2/Problem5.c
/*      PROBLEM 5
 * If value of an angle is input through the keyboard,
 * write a program to print all its trigonometric ratios.
 */

// include the libraries
#include <stdio.h>
#include <math.h>

int main()
{
    // declare variables and take input
    float a;
    printf("Enter angle: \n");
    scanf("%f", &a);

    // calculate trigonometric ratios:

    // Sine:
    printf("Sine of angle %.3f is %.3f\n", a, sin(a));

    // Cosine:
    printf("Cosine of angle %.3f is %.3f\n", a, cos(a));

    // Tangent:
    printf("Tangent of angle %.3f is %.3f\n", a, tan(a));

    // Cotangent:
    printf("Cotangent of angle %.3f is %.3f\n", a, (1 / tan(a)));

    // Secant:
    printf("Secant of angle %.3f is %.3f\n", a, (1 / cos(a)));

    // Cosecant:
    printf("Cosecant of angle %.3f is %.3f\n", a, (1 / sin(a)));
}

File: Chapter-2/Java/Problem5.java

//import libraries
import java.util.Scanner;

public class Problem5 {
    public static void main(String[] args) {
        double a;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the angle: ");
        a = sc.nextDouble();

        System.out.println("The trigonometric ratios of angle" + a + "are: ");
        System.out.printf("SINE of angle a is : %.3f\n", Math.sin(a));
        System.out.printf("COSINE of angle a is : %.3f\n", Math.cos(a));
        System.out.printf("TANGENT of angle a is : %.3f\n", Math.tan(a));
        System.out.printf("COTANGENT of angle a is : %.3f\n", 1 / Math.tan(a));
        System.out.printf("SECANT of angle a is : %.3f\n", 1 / Math.cos(a));
        System.out.printf("COSECANT of angle a is : %.3f\n", 1 / Math.sin(a));

        sc.close();
    }
}
